README.md

```diff
@@ -59,6 +59,28 @@ Note: Argument [quality] has a default value of 0 which corresponds to
 220-260 kbit/s VBR encoding.<br> 
 See [this article](https://wiki.hydrogenaudio.org/index.php?title=LAME#Recommended_encoder_settings) for more information.
 
+### flacer
+A script converting various audio files into a standard 16bit/44.1kHz flac format.<br>
+Usage:
+> flacer [filepath]
+
+Arguments:
+- filepath: path to the audio file to be converted
+
+### flactag
+Tool for metadata addition or editoin of flac filesm, automatically naming the file:
+>[Artist] - [Title]<br>
+
+Usage:
+> flactag [filepath of flac] [filepath of image]
+
+Arguments:
+- filepath of flac: location of the audio file
+- filepath of image: location of the image that will be set as an album conver. It can be both of jpg and png formats and have any aspect ratio.
```

Comment:
Argument description should provide all the necessary information, instead of trying to contain it in argument's name.